ppl/src/main/antlr/OpenDistroPPLLexer.g4

```diff
@@ -137,20 +137,29 @@ DC:                                 'DC';
 ABS:                                'ABS';
 CEIL:                               'CEIL';
 CEILING:                            'CEILING';
+CONV:                               'CONV';
+CRC32:                              'CRC32';
 EXP:                                'EXP';
 FLOOR:                              'FLOOR';
 LN:                                 'LN';
 LOG:                                'LOG';
 LOG10:                              'LOG10';
 LOG2:                               'LOG2';
+MOD:                                'MOD';
+POW:                                'POW';
+POWER:                              'POWER';
+ROUND:                              'ROUND';
+SIGN:                               'SIGN';
+SQRT:                               'SQRT';
+TRUNCATE:                           'TRUNCATE';
 
 // LITERALS AND VALUES
 //STRING_LITERAL:                     DQUOTA_STRING | SQUOTA_STRING | BQUOTA_STRING;
 ID:                                 ID_LITERAL;
 INTEGER_LITERAL:                    DEC_DIGIT+;
 DECIMAL_LITERAL:                    (DEC_DIGIT+)? '.' DEC_DIGIT+;
 
-fragment ID_LITERAL:                [A-Z_$0-9@]*?[A-Z_$\-]+?[A-Z_$\-0-9]*;
+fragment ID_LITERAL:                [A-Z_]+[A-Z_$0-9@\-]*;
```

Comment:
The original `ID_LITERAL` regex allows digital numbers and special char `@`, `\-` at the starting place. The negative numbers are also recognized as IDs when I tested out some cases in functions. So I changes the rule to only allow letters and underscore(`_`) as the start character.

Reply:
Sure, I will check this issue too. Because I'm changing this to define what is identifier, will see if digit should be not allowed as prefix. You can keep this change in your PR.